pootle/templates/admin/xhr_paid_tasks_report.html

```diff
@@ -18,13 +18,13 @@
       <%
         var amount = row.amount;
```

Comment:
OK, but s that really what was intended?  Sounds more like it fits in with the mistake of using concatenation.  I'd rather see an else clause to handle that case more intentionally.